### DISCUSSION TITLE
Event Webhooks

### DISCUSSION THREAD

**Discussion by lpbonomi** [SoftwareBrothers/admin-bro#1522] [Ideas]
It would be cool to be able to call a webhook when a certain action (e.g. update a record) is done. This could be very useful for sites that have static files that depend on these records, for example, [NextJS](https://nextjs.org/docs/app/building-your-application/data-fetching/revalidating)

**Comment by dziraf** [comment]
You should be able to create an `after` hook which does exactly that though?
